migrations/2021-02-28-162616_clean_empty_post_urls/up.sql

```diff
@@ -0,0 +1 @@
+UPDATE post SET url = NULL where url = '';
```

Comment:
Make sure `diesel migration redo` ( or `diesel migration revert` ) works... iirc the `down.sql` cannot be empty, it must have a dummy statement like `select * from user_`.... or anything.